src/App.tsx

```diff
@@ -15,36 +23,82 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function sortGoods(goods: string[], sortField: SortType, reversed: boolean) {
+  const sortedGoods = [...goods].sort((good1, good2) => {
+    switch (sortField) {
+      case SortType.Alphabet:
+        return good1.localeCompare(good2);
+      case SortType.Length:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  });
+
+  return reversed ? sortedGoods.reverse() : sortedGoods;
+}
```

Comment:
The `sortGoods` function is implemented correctly, but it could be optimized by avoiding the creation of a new array with `reverse()` if `reversed` is false. Consider returning `sortedGoods` directly when `reversed` is false.